crates/ark/src/data_explorer/summary_stats.rs

```diff
@@ -155,11 +161,19 @@ fn empty_column_summary_stats() -> data_explorer_comm::ColumnSummaryStats {
     }
 }
 
-fn get_stat<T: Clone>(stats: &HashMap<String, T>, name: &str) -> anyhow::Result<T> {
+fn get_stat<Return, T: Clone>(stats: &HashMap<String, T>, name: &str) -> anyhow::Result<Return>
+where
+    Return: TryFrom<T>,
+{
     let value = stats.get(name);
 
     match value {
-        Some(value) => Ok(value.clone()),
+        Some(value) => {
+            let value: Return = unwrap!(value.clone().try_into(), Err(_) => {
+                return Err(anyhow!("Can't cast to return type."))
```

Comment:
```suggestion
            let value: Return = unwrap!(value.clone().try_into(), Err(_) => {
                return Err(anyhow!("Can't cast to return type: {err:?}"))
```

We don't care about the reason here?